.github/mergify.yml

```diff
@@ -177,41 +177,3 @@ pull_request_rules:
           - branch-perf-v16
         assignees:
           - "{{ author }}"
-  - name: Automate backport pull request manager-3.2
```

Comment:
Why just removing ? you want want to add 3.4 ?

Reply:
We agreed with the team that starting from release 3.4 we stop supporting the separate release branch for Manager and keep only master. For the last 9 months I'm working with Manager, there were no cases we were needed to execute different tests for different Manager versions. So, keeping two branches and constant backporting (+ possible conflicts resolution) seems redundant.

Reply:
@roydahan FYI

it's similar to what we did for scylla-operator in SCT, we stopped branching at some point.

what would you do for minor releases ? still run tests from master ? 
that would require much more attention to backward compatibility in the tests, which isn't that great now.

Reply:
Yes, run everything from master.

Actually, I prefer the approach with separate branches because sometimes it might come in handy (it wasn't a case last time as I mentioned above but it may appear).
The only concern and reason we decided to switch, conflicts resolution during backporting. I got into such cases with previous 3.3 release where the `manager-3.3` branch was missing some commits from master on which my solution was based. I spent some time figuring out what I should backport to make it work but failed and left the branch outdated (if I'm not mistaken it was related to Argus charts).

Since I can't keep an eye on every PR which is being merged into master on the subject of missing backport manager label, we have to switch to this approach with single branch. If there's ever a need for a separate branch, we can always introduce it again.

Reply:
when you introduce it, it hard to pick the right commit to cut from
cause you might be missing things related to manager, or to SCT stability on the other hand.

I guess with manager is relatively ease, cause the volume of related commits to manager is quite small.